elements/rh-footer/docs/30-code.md

```diff
@@ -1,20 +1,90 @@
-{% renderInstallation %}{% endrenderInstallation %}
+{% renderInstallation %}
+~~~html
+<link rel="stylesheet" href="https://redhatstatic.com/dx/v1-alpha/@rhds/elements@1.0.2/rh-footer/rh-footer-lightdom.css">
+~~~
+Lightdom CSS is required to ensure a reduced [Cumulative Layout Shift (CLS)](https://web.dev/cls/) experience before the component has fully initialized.
+{% endrenderInstallation %}
 
-## Usage
-  {% playground tagName=tagName %}{% endplayground %}
-  {% cta href="./demo/", target="_blank" %}
-View the demo in a new tab
-  {% endcta %}
+## &lt;rh-footer&gt;
 
-{% renderSlots %}{% endrenderSlots %}
+    {% renderSlots for='rh-footer', level=3 %}{% endrenderSlots %}
```

Comment:
Wonder if it would be worth it to create a shortcode for any time we're going to want a specific component to be shown in the docs?  Like {% renderSubcomponent tag='rh-footer' %} {% endrenderSubcomponent %}? Because I see accordion and secondary nav already doing something similar to this

Reply:
I like this idea. Or, even better, a single short code where a list of parts are given?

Reply:
Can we split this out into a different issue?

Reply:
Done #1073